### DISCUSSION TITLE
Nach wechsel von hm-mod-rpi-pcb auf Homematic IP USB-Stick keine Einstellung der Geräte mehr möglich

### DISCUSSION THREAD

**Discussion by ElGauchooooo** [OpenCCU/OpenCCU#2178] [🇩🇪 Fragen&Antworten] [answered]
### Describe the issue you are experiencing

Ich hatte bis gestern RaspberryMatic als HomeAssistant-Addon auf einem RPI 4 mit einem hm-mod-rpi-pcb installiert (FW Version 3.65.8.20220831).

Habe nun HomeAssistant auf einen NUC umgezogen, Backup gemacht und wieder eingespielt. Ebenfalls habe ich statt des hm-mod-rpi-pcb auf einen Homematic IP USB-Stick gewechselt. RaspberryMatic erkennt zwar alle Geräte und bekommt auch Änderungen am Gerät selbst mit (z.B. Thermostat Temperatur von 23->22 Grad ändern oder Fensterkontakt Fenster öffnen / schließen), allerdings sind keine Änderungen mehr aus HomeAssistant / RaspberryMatic selbst möglich. 

Wenn ich z.B. die Temperatur in RaspberryMatic von 22->23 Grad zurück ändere, bekomme ich kurz darauf "Device communication disturbed" als ServiceMessage angezeigt. Nach einigen Minuten verschwindet die Meldung wieder, die Änderung wurde aber nicht vollzogen. 

Den Wechsel habe ich gestern Abend durchgeführt, seit dem sind ~10 Stunden vergangen.

### Describe the behavior you expected

Kommunikation in beide Richtungen funktional.

### Steps to reproduce the issue

s.o.

### What is the version this bug report is based on?

3.65.8.20220831

### Which base platform are you running?

intelnuc (intel NUC)

### Which HomeMatic/homematicIP radio module are you using?

HmIP-RFUSB

### Anything in the logs that might be useful for us?

```txt
Mounting /data as /usr/local (Home Assistant Add-On): OK
Identifying host system: oci, OK
Initializing RTC Clock: onboard, OK
Running sysctl: OK
Checking for Factory Reset: not required
Checking for Backup Restore: not required
Initializing System: OK
Setup ca-certificates: OK
Starting logging: OK
Init onboard LEDs: init, OK
Starting irqbalance: OK
Starting iptables: OK
Starting network: eth0: link up, fixed, firewall, inet up, 172.30.33.3, OK
Identifying Homematic RF-Hardware: ....HmRF: n/a, HmIP: HMIP-RFUSB-TK/eQ-3 HmIP-RFUSB@usb-0000:01:1b.0-1, OK
Updating Homematic RF-Hardware: HMIP-RFUSB-TK: 4.2.14, not necessary, OK
Starting hs485dLoader: disabled
Starting xinetd: OK
Starting eq3configd: OK
Starting lighttpd: OK
Starting ser2net: disabled
Starting ssdpd: OK
Starting ha-proxy: OK
Starting NUT services: disabled
Initializing Third-Party Addons: OK
Starting LGWFirmwareUpdate: ...OK
Setting LAN Gateway keys: OK
Starting hs485d: disabled
Starting multimacd: not required
Starting rfd: no BidCos-RF hardware found
Starting HMIPServer: ........OK
Starting ReGaHss: .OK
Starting CloudMatic: OK
Starting NeoServer: disabled
Starting Third-Party Addons: OK
find: /usr/local/etc/monit*.cfg: No such file or directory
Starting crond: OK
Setup onboard LEDs: booted, OK
Finished Boot: 3.65.8.20220831 (raspmatic_oci_amd64)
```

**Comment by jens-maus** [comment, marked as answer]
1. Du schickst hier einen Bugreport auf Basis einer alten (3.65.8.20220831) Version (sollte immer auf der aktuellsten Version basieren!)
2. Du nutzt ein `HmIP-RFUSB-TK` (man beachte das TK) welches suboptimal ist bzgl. fehlender Anzeige des CarrierSense und der nicht vorhandenen Anbindung an ein `HmIP-HAP` bzw. `HmIPW-DRAP` und dafür wird es auch in Zukunft keine Firmware-Updates mehr geben. Kauf den hier: https://de.elv.com/elv-homematic-ip-arr-bausatz-rf-usb-stick-fuer-alternative-steuerungsplattformen-hmip-rfusb-fuer-smart-home-hausautomation-152306
3. Da du keine Anzeige bzgl. CarrierSense (Weil du eben ein HmIP-RFUSB-TK statt eines HmIP-RFUSB (ohne TK) nutzt) hast du auch keine Idee wie stark die Störsignale in der näheren Umgebung des USB Sticks sind.
4. Du sagst nicht ob du den USB Stick wie vorgeschlagen via 2-3 m langem USB Verlängerungskabel und zusätzlich einem USB Entstörfilter (https://de.elv.com/elv-usb-entstoerfilter-usb-ef1-komplettbausatz-152745) nutzt. Beides ist stark angeraten!
5. Du hast in deiner Problembeschreibung nicht angegeben das du nicht RaspberryMatic direkt auf einem intelNUC nutzt sondern wohl innerhalb von HomeAsisstant das auf dem intelNUC (entweder native oder virtuell) läuft. Beides ist aber für Problemanalysen relevant.
6. Ist dies kein Bug, sondern ein ein lokales Nutzungsproblem deinerseits und gehört daher hier nicht her sondern in ein Diskussionsforum wie hier in GitHub oder homematic-forum.de. Deshalb habe ich dein Issue-Ticket mal in den Diskussionsteil verschoben.
